build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.CS.targets

```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
     <PropertyGroup>
-        <DefineConstants
+        <DefineConstants>
```

Comment:
Shouldn't this entire line be deleted?

Reply:
Good question. Taking a closer look, seems like we don't even need this line.

Reply:
@DrusTheAxe Want to double check with you to see if we can remove it.

Reply:
Correct -- delete line 4. That's a typo

For valid sample syntax see https://stackoverflow.com/questions/36793499/defineconstants-in-addition-to-existing-ones-in-csproj#36793713
![image](https://user-images.githubusercontent.com/72222/161623340-d02e2fe0-1375-4280-8fb0-294751bfdb37.png)